tests/cpp_random_unit.cpp

```diff
@@ -378,7 +379,7 @@ DEFINE_TEST(random_doublecheck_test_64) {
         const Roaring64Map &right = roars[rand() % NUM_ROARS];
 
 #ifdef ROARING_CPP_RANDOM_PRINT_STATUS
-        printf("[%lu]: %llu %llu %llu\n", step,
+        printf("[%lu]: %" PRIu64 " %" PRIu64 " %" PRIu64 "\n", step,
```

Comment:
In general of course this is portable.  However, MSVC didn't support C99 until the MSVC 2013 release, 9 years ago (and actually even that release didn't support all of it).  Maybe we don't care enough about supporting older MSVC and that's fine.

Reply:
We require C11 in CRoaring.